scaffolding-chef-inspec/lib/scaffolding.ps1

```diff
@@ -74,7 +79,7 @@ if(!`$env:CFG_CHEF_LICENSE){
 `$PROFILE_PATH="{{pkg.path}}/{{pkg.name}}-{{pkg.version}}.tar.gz"
 
 function Invoke-Inspec {
-    inspec exec `$PROFILE_PATH --config `$CONFIG --log-level `$CFG_LOG_LEVEL
+    inspec exec `$PROFILE_PATH --json-config `$CONFIG --log-level `$env:CFG_LOG_LEVEL
```

Comment:
`--json-config` is deprecated and will dissappear in a future version of InSpec, please use `--config`

Reply:
This is due to the version of InSpec

Reply:
Let's make sure we add a # TODO: comment there, just to make sure we're solid with a future version (when it arrives)